pull_request_test.rb

```diff
@@ -28,16 +28,19 @@ def call
   end
 end
 
+token = ENV["OAUTH_TOKEN"] || File.read("OAUTH_TOKEN").chomp
```

Comment:
The idea here was that it's a little easier IMO to manage this file in a local file rather than an environment variable in my global dotfiles or in 1password, kind of like our `.env.development` files in other repos. I couldn't actually figure out how to use this file to test anything, so I'm likely to just undo these changes, unless I can figure out how to use this.